clientSide/capstone_project/src/components/Signin.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import "./Signin.css"
+import { useState } from "react";
+function DisplaySignin()
+{
+
+    const [username,setUsername]=useState("");
+    const [password,setPassword]=useState(""); 
+    
+    function clickMe()
+    {
+        let res=JSON.parse(localStorage.getItem("studentsignindata")) || [];
+       // let results=JSON.parse(localStorage.getItem("studentdata"));
+        if(username!=="" && password!=="")
+        {
+            localStorage.setItem("studentsignindata",JSON.stringify([...res,{username,password}])); 
+            // for(let i=0;i<results.length;i++)
```

Comment:
remove commented code and format the document with prettier-code-formatter available in vscode extensions